### DISCUSSION TITLE
2022/12/17/DoYouLoveWhatYouDo

### DISCUSSION THREAD

**Discussion by giscus** [KMIT-Students/blog-demo#18] [General]
# 2022/12/17/DoYouLoveWhatYouDo

Have you ever enjoyed doing something so much that you NEVER feel the amount of time you invest in it as hard work ? This is called passion. The “something” might be a sport, playing an instrument, learning a subject that you love or anything. Unfortunately many of us in India don’t have the luxury to follow our passion. Instead we are pressured to be a doctor, an engineer or some line of work that involves a lot of money.

https://kmit-anubhav.github.io/2022/12/17/DoYouLoveWhatYouDo.html

**Comment by creator318** [comment]
Very well written.
Also a perfect topic...

**Reply by MaruthiCharan2403** [reply]
yup

**Comment by MaruthiCharan2403** [comment]
In my opinion, these things are easy to say but very hard to apply in life. Society should change in order to follow our passion

**Reply by MaruthiCharan2403** [reply]
overall its wonderful
